projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/cast/LLVMToI8Node.java

```diff
@@ -132,6 +132,11 @@ public byte executeI8(LLVM80BitFloat from) {
     public abstract static class LLVMAddressToI8Node extends LLVMToI8Node {
```

Comment:
Ah I filled in the I8 and I32 ones by default, but I only need the I64 one. I'll remove the others.

I'm also trying the `LLVMTo*Node` idea now.